app.json

```diff
@@ -36,10 +37,19 @@
       "output": "static"
     },
     "plugins": [
-      "expo-router"
+      "expo-router",
+      "expo-font"
     ],
     "experiments": {
       "typedRoutes": true
+    },
+    "extra": {
+      "router": {
+        "origin": false
+      },
+      "eas": {
```

Comment:
Is this an accidental add, or did you mean to add an eas project to the project?